server/plugins/graphql/SlackPlugin.graphql

```diff
@@ -0,0 +1,3 @@
+Type SlackConfig {
```

Comment:
This is a good start, but I don't think it's used anywhere. Also should `Type` be lowercase?
I think in the next pass we should have every plugin define their own GraphQL snippet that'll get stitched together into the main schema (to keep everything clean & separated).

Reply:
Hence the neglected comment in the main graphql. I'll work on this (I see the issue)